frontend/components/elements/StoryItem.vue

```diff
@@ -34,22 +34,44 @@
                     <VIcon name="arrow-down"/>
                 </BButton>
 
-                <span class="js-drag-drop ml-2" v-if="isView(['backlog', 'planning-sprint'])" title="Drag to change order">
-                    <VIcon 
-                        name="arrows" 
-                        label="Drag and drop to change order"/>
+                <span
+                    :class="{ 'js-drag-drop': sortable, 'ml-2': true }"
+                    v-if="isView(['backlog', 'planning-sprint'])"
+                    @dblclick="!sortable || startKeyboardSort()"
+                    :title="sortable ? 'Drag to change order, double click to use keyboard' : 'Sort by position ascending (default order) to be able to change order'">
+                    <nobr>
+                        <VIcon
+                            name="arrows"
+                            label="Drag and drop to change order"
+                            :class="{ [$style.fade]: !sortable }"/>
+
+                        <strong>{{ position }}</strong>
+                    </nobr>
                 </span>
-                <!--{{ data.position }}-->
+
+                <strong v-if="isView('planning-backlog')">
+                    {{ position }}
+                </strong>
             </td>
 
+            <td v-if="isView(['sprint', 'history'])">
+                <strong>
+                    {{ position }}
+                </strong>
+            </td>
             <!-- Identifier -->
             <td>
                 {{ data.identifier }}
             </td>
 
             <!-- Story -->
-            <td>
+            <td @click="expandedView = !expandedView">
                 {{ data.title }}
+
+                <p v-show="expandedView" class="mt-2">
+                    Notes: <br>
+                    {{ data.description || '(no notes)' }}
```

Comment:
I cannot see, that I am able to click here. I needed to search 
for the right area to click. So maybe it would be a good idea to format the title a bit different ... like a link?

Reply:
I've put a little arrow there now